### ISSUE TITLE
What do you like about working in open source?

### ISSUE THREAD

**Issue by cassiemc** [bwinton/ama#4]

**Comment by bwinton** [comment]
There are a couple of different things that I like.

On the personal side, when I applied for a job at Mozilla Messaging (more than six years ago!), @davidascher asked me for some sample code.  That turned out to be a fairly large problem, because the company I had worked for the previous seven years had gone out of business, and there wasn’t anyone left working there who I could ask for permission to send the code from.  These days, all my work goes into [bugzilla](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=ALL%20assignee%3Abwinton&list_id=12379163) or on [github](https://github.com/bwinton), so the next time someone asks, I’ll have a wealth of material to give them!

On the societal side, I like that people can take the stuff I make, and learn techniques from it, and/or make changes and pull requests that will make it better for everyone!  I’ve written a lot of tools that could make people’s lives easier, but are locked up at companies who may or many not exist anymore, and that always makes me a little sad.
